### PR TITLE
Added more tests for nuget.exe list to handle the following

### PR DESCRIPTION
**Pull request by deepakaravindr**
1) v3 package source without an entry for package publish endpoint
2) v3 package source which is not available

@yishaigalatzer @emgarten @MeniZalzman @feiling

### CODE REVIEW

File: src/NuGet.CommandLine/Commands/ListCommand.cs [hunk 1]
```diff
@@ -124,13 +124,26 @@ private IPackageRepository GetRepository(IEnumerable<string> listEndpoints)
...
                 if (listEndpoint != null)
                 {
                     listEndpoints.Add(listEndpoint);
                 }
+                else
+                {
+                    var message = string.Format(
+                        LocalizedResourceManager.GetString("ListCommand_ListNotSupported"),
+                        packageSources[i].Source);
+                    Console.LogWarning(message);
```
Comment (emgarten) [inline src/NuGet.CommandLine/Commands/ListCommand.cs:145]: Add a new line above Console.LogWarning

**Comment by emgarten** [comment]
:shipit:

**Comment by MeniZalzman** [comment]
:shipit:

**Comment by deepakaravindr** [comment]
Merged with https://github.com/NuGet/NuGet.PackageManagement/commit/93c1e3eb02d47526a03df751c14bb05fbf278516

**Pull request closed without merging**